media-video/qmplay2/metadata.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>reagentoo@gmail.com</email>
+		<name>Dmitry Baranov</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<upstream>
+		<remote-id type="github">zaps166/QMPlay2</remote-id>
+	</upstream>
+	<use>
+		<flag name="animeodcinki">Build with AnimeOdcinki support</flag>
```

Comment:
I suspect you may not like this question but do you really need that many USE flags? Given that you don't enable any of them by default, you're asking the user to do a lot of flag toggling to get the things he may want. If those things don't have a specific reason to be toggle-able (e.g. being large, having external dependencies or causing some side effects like doing random stuff automatically when enabled), you can just enable most of it by default.

Reply:
Should I reduce flags count or enable most flags by default? I use this player with only 4 flags.

Reply:
My suggestion would be to reduce the number of flags, and force on things where it doesn't really make sense to disable stuff.